### ISSUE TITLE
Maybe add a install guide?

### ISSUE THREAD

**Issue by tvvocold** [zirafe/zirafe#22]
I try to install it in openshift but it didn't work, i had mkdir -p /var/zirafe/{files,links,trash} but the links allways got wrong path.I think the reason maybe apache.So if any guide or .htaccess is helpful.

**Comment by edausque** [comment]
Yes, for this current version of zirafe is only working with nginx.
Indeed, rewriting is mandatory for zirafe. Rewriting rules are currently inside of nginx configuration, so if you are using apache, it won't work until you create the same rules for apache.
